docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -3,115 +3,85 @@ id: user-logout
 title: User logout
 ---
 
-Ory Identities supports two logout flows:
+```mdx-code-block
+import { apiFlow } from "./code/logout"
+import CodeTabs from "@theme/Code/CodeTabs"
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import browserExample from "!!raw-loader!./code/logout/samples/browser/node.ts"
+import spaExample from "!!raw-loader!./code/logout/samples/spa/react.txt"
+import CodeBlock from "@theme/CodeBlock"
+```
 
-- Browser-based (easy): This flow works for all applications running on top of a browser. Websites, single-page apps,
-  Cordova/Ionic, and so on.
-- API-based (advanced): This flow works for native applications like iOS (Swift), Android (Java), Microsoft (.NET), React Native,
-  Electron, and others.
+The logout flow is used to terminate an Ory Identities session. Ory supports logout on all supported clients:
 
-import { apiFlow, browserFlow } from "./code/logout"
-import CodeTabs from "@site/src/theme/Code/CodeTabs"
+- [Server rendered browser applications](#server-side-browser-applications)
+- [Client side browser applications](#client-side-browser-applications)
+- [Native applications](#native-api-applications)
 
-## Self-service logout for server-side browser applications
+## Server-side browser applications
 
-To ensure that a logout was indeed intended by the user, Ory Identities first generates a Logout URL for a given Ory Session
-Cookie. You can then open the Logout URL in the Browser.
+To ensure that a logout was intended by the user, Ory Identities first generates a logout URL for a given Ory Session cookie. You
+can then open the logout URL in the browser.
 
-import SelfServiceLogoutBrowserFlow from "@site/src/theme/SelfServiceLogoutBrowserFlow"
+The logout URL is valid for the duration of the session. This means that you can call the endpoint once and cache this URL for
+some time before you call the endpoint again.
 
-<SelfServiceLogoutBrowserFlow />
+Alternatively, you can create the logout URL only when the user actually clicks the logout button and immediately call the
+resulting `logout_url` from your server to log out the user. Beware, that this method can lead to inconsistent behavior for users
+that switch accounts often.
 
-After successful logout, the browser will be redirected either to the `return_to` query parameter from the initial request URL, or
-fall back to the `default_browser_return_url` value set in Ory Identities' configuration file:
-
-```yaml
-selfservice:
-  flows:
-    logout:
-      after:
-        default_browser_return_url: http://test.ory.sh:4000/
+```mdx-code-block
+<CodeBlock language="ts" title="index.ts">{browserExample}</CodeBlock>
 ```
 
-<CodeTabs items={browserFlow} />
-
-If an error occurs, the browser is redirected to the [Error UI](./user-facing-errors.md).
-
-## Self-service logout for client-side browser applications
+### After logout redirect
```

Comment:
```suggestion
### Post-logout redirect
```

Reply:
It's called `After-logout redirect` in the cosnole, best to stay consistent here, I think.

Reply:
Hm, it's not - just checked

<img width="648" alt="image" src="https://user-images.githubusercontent.com/29116492/206458361-311e2342-1de7-420d-a7fe-08293f8e2354.png">

Reply:
Hm, wat? I swear it was a day ago. Alright then.